src/acceptance/administrator/components/com_users/UserNotesCest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * @package     Joomla.Tests
+ * @subpackage  Acceptance.tests
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace administrator\components\com_users;
+use Page\Acceptance\Administrator\UserNotesListPage as UserNotesList;
+use Page\Acceptance\Administrator\UserNotesFormPage as UserNotesForm;
```

Comment:
Why do we use aliases here?

Reply:
Aliases are very helpful as when I was asked to change the name of file from userNotesListPage to UserNotesListPage. I only had to make the change once

Reply:
Thats not a good reason to use aliases. 
Aliases reduce readability, so we should only use them in case of class name conflicts.

Reply:
Ok. I have removed aliases in my latest commit